ndohyep/templates/core/article_page.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+{% extends "core/_article_page.html" %}
```

Comment:
Peculiar, ok, let's leave it as is then and I'll check with @miltontony to see if we made these changes or not.